clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -395,15 +395,24 @@ public long getMaxElapsedTimeMSSinceLastPollInConsumerPool() {
          * are zero-based and ConsumptionTasks are submitted to the executor in order.
          */
         Thread slowestThread = threadFactory.getThread(slowestTaskId);
-
+        SharedKafkaConsumer consumer = consumerToConsumptionTask.getByIndex(slowestTaskId).getKey();
+        Map<PubSubTopicPartition, TopicPartitionIngestionInfo> topicPartitionIngestionInfoMap =
+            getIngestionInfoFromConsumer(consumer);
+        // Convert Map of ingestion info for this consumer to String for logging with each partition line by line
+        StringBuilder sb = new StringBuilder();
+        for (Map.Entry<PubSubTopicPartition, TopicPartitionIngestionInfo> entry: topicPartitionIngestionInfoMap
+            .entrySet()) {
+          sb.append(entry.getKey().toString()).append(": ").append(entry.getValue().toString()).append("\n");
```

Comment:
BTW, how big this log could be for a multi-tenant cluster? And how often would this log be printed?

Reply:
This is controller by:

Reply:
This is controller by: 
`(!REDUNDANT_LOGGING_FILTER.isRedundantException(slowestTaskIdString)) {`

Once per minute.

Reply:
Synced up with Hao offline; according to the metrics, this log line could be as long as 200 lines. Updated the interval from 1 minute to 10 minutes.